packages/cli/test/utils/simulation/cl_clients/lodestar.ts

```diff
@@ -0,0 +1,181 @@
+import {mkdir, writeFile} from "node:fs/promises";
+import {dirname, join} from "node:path";
+import {Keystore} from "@chainsafe/bls-keystore";
+import {nodeUtils} from "@lodestar/beacon-node/node";
+import {LogLevel} from "@lodestar/utils";
+import {IBeaconArgs} from "../../../../src/cmds/beacon/options.js";
+import {IValidatorCliArgs} from "../../../../src/cmds/validator/options.js";
+import {IGlobalArgs} from "../../../../src/options/globalOptions.js";
+import {JobOptions, CLClientGenerator, CLClientOptions, Runner, RunnerType, Job} from "../interfaces.js";
+import {callHttp, LODESTAR_BINARY_PATH} from "../utils.js";
+
+export const generateLodeStarBeaconNode: CLClientGenerator = (opts: CLClientOptions, runner: Runner): Job => {
```

Comment:
small thing, but we usually treat lodestar as a single word.

Lodestar
lodestar

not 

LodeStar
lodeStar
lode_star